app/routes/reports.tsx

```diff
@@ -63,22 +88,24 @@ export default function Reports() {
 
 			<ReportsHeader reports={reports} amounts={contributionAmounts.amounts} />
 
-			<section className="grid grid-rows-1 md:grid-cols-2 xl:grid-cols-3 2xl:grid-cols-4 gap-5 md:gap-3">
-				{reports.map((report: Report) => (
-					<Link to={`/reports/${report.slug}`} key={report.hypercertId}>
-						<ReportCard
-							hypercertId={report.hypercertId}
-							image={report.image}
-							title={report.title}
-							summary={report.summary}
-							category={report.category}
-							state={report.state}
-							totalCost={report.totalCost}
-							fundedSoFar={report.fundedSoFar}
-						/>
-					</Link>
-				))}
-			</section>
+			{reports.length && (
+				<section className="grid grid-rows-1 md:grid-cols-2 xl:grid-cols-3 2xl:grid-cols-4 gap-5 md:gap-3">
+					{reports.map((report: Report) => (
```

Comment:
--> **safeReports**.map((report: Report)

Reply:
actually we don't need safeReports at all anymore. It was a fix for a bug that no longer happens afaik